UKHomeOffice/sgmr-service
# Update arrival default unlocode to ZZZA

### AC / Description of changes
Discovered arrival default unlocode was set to ZZZD, which is incorrect and was causing CBP to reject the submitted report as departure and arrival cannot have the same unlocode.


### To Test
- Submit a report with an departure and arrival port both of which do not have unlocodes
- Run Cypress reports, reports should pass


### Notes

---

**Remember to merge to feature branches not `master`.**
